content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+@griffiths_language_2007 were the first to show that the analytical structure of iterated learning with uniform Bayesian agents can correspond, depending on the way the agents produce new iterations, to well-known statistical inference methods (Gibbs sampling and a flavour of the EM algorithm).
+In such a setup, iterated learning predictably converges towards distributions determined by the internal prior distributions of agents (i.e. their inference bias).
+As a consequence, in those analytically derived situations one can straightforwardly predict the final distributions that should evolve under iterated learning, a fact that @kalish_iterated_2007 verified with humans in a function learning task.
+@griffiths_using_2008 further exploit this result by using it in the reverse direction:
+since the outcome of iterated learning, for specific setups, is predictable on the basis of participants' priors, one can use such experiments to investigate the inductive biases of participants.
+The authors confirmed this, showing that the method infers well-known participant biases in category learning tasks.
+@griffiths_theoretical_2008 then explored the relevance of these findings for the study of cultural evolution, showing in particular that individual cognitive biases can have significant effects on long-term cultural evolution.
+@reali_evolution_2009 further related those results to the evolution of vocabulary, showing that they are consistent with experimental cases of word-meaning mapping regularisation.
+Finally @perfors_language_2014, through analytical derivation and experimental confirmation, reintroduced the impact of the external world in those results;
+the authors showed that the structure of referents (i.e., the external world) will also play a role in the final evolved language, provided it has an effect on the choice of items people actually talk about (versus, the choice of items talked about only depends on the language itself).
+
+@scott-phillips_language_2010 and @tamariz_cultural_2016 provide further reviews of the iterated learning literature, and @galantucci_experimental_2012 and @roberts_investigating_2017 offer reviews of experimental semiotics.
+An interesting and important development in recent works is the reintroduction of pragmatics into theoretical questions.
+@scott-phillips_pragmatics_2017, in particular, reaffirms the central role of pragmatics in the creation and understanding of meaning in context, and argues for a much stronger focus on the evolution of pragmatics itself, that is, as he envisions it, on the evolution of *ostensive communication*.
+Let me close this review by noting that there is an increasing convergence both in the literature and in empirical questions, of the cultural evolution and language evolution fields;
+the intersection of questions from the two fields is likely to push theoretical issues forward.
+
+\add{sperber-beyond-2015}
+
+
+#### Digital media
```

Comment:
indeed! Tu as des pointeurs pour les travaux majeurs que j'ai manqués par chance?

TODO
- [ ] insist on transmission without change, i.e. really epidemiological (as Cogito, and Acerbi, note too)
- [ ] introduce Granovetter's Threshold Model
- [ ] introduce the independent Cascade model
- [ ] then online social media
- [ ] then focus on CSS/CplxSys, online and offline, i.e. mostly through computing power and massive data collection